pkg/pool-manager/pool_test.go

```diff
@@ -414,4 +416,75 @@ var _ = Describe("Pool", func() {
 		})
 	})
 
-})
+	Describe("Multus Network Annotations API Tests", func() {
+		BeforeEach(func() {
+			poolManager := createPoolManager("02:00:00:00:00:00", "02:00:00:00:00:01", &vmConfigMap)
+			Expect(poolManager).ToNot(Equal(nil), "should create pool-manager")
+
+			Context("When pool-manager is configured with available addresses", func(){
```

Comment:
Context without any content (except other contexts) is suspicions. If you are not setting anything, you are probably keeping the context implicit and defined in internal code. That is risky since it may get outdated quickly, it does not keep the context info visible and does not force us to actually follow the context.

I think that in this case, this Context is redundant since you describe the addresses parameter in the following Context. Please drop it.

Reply:
done

Reply:
@ormergi I don't see it being changed. There are still 2 contexts, the outter does not set anything.